extensions/src/ACRE2Arma/signal/antenna/antenna.cpp

```diff
@@ -30,6 +30,9 @@ float32_t acre::signal::antenna::gain(const glm::vec3 dir_antenna_, const glm::v
     if ((f_ < _min_frequency) || ((f_ + _frequency_step) > _max_frequency)){
         return -1000.0f;
     }
+    if (glm::length(dir_signal_) <= 0.0f) {
+        return 0.0f; // cannot get direction if antennas have same position
```

Comment:
That is not "really true". You can have the same position but different angles but that would mean reworking some part of the code. However this is out of the scope of this PR.